rust/processor/src/config.rs

```diff
@@ -26,13 +26,21 @@ pub struct IndexerGrpcProcessorConfig {
     pub db_pool_size: Option<u32>,
     #[serde(default = "IndexerGrpcProcessorConfig::default_gap_detection_batch_size")]
     pub gap_detection_batch_size: u64,
+    #[serde(default = "IndexerGrpcProcessorConfig::default_pb_channel_txn_chunk_size")]
+    pub pb_channel_txn_chunk_size: usize,
```

Comment:
Do we also wanna make a config var for 
* DB max pool size (does that even make sense to have diff max pool size for different processors?)
* # of concurrent write to DB tasks